plat/common/bootinfo.lds.S

```diff
@@ -0,0 +1,11 @@
+/* Define .uk_bootinfo section that depending on the boot protocol can
+ * be populated by with boot information
+ */
+SECTIONS {
+	. = ALIGN(8);
+	.uk_bootinfo :
+	{
+		KEEP(*(.uk_bootinfo))
+	} :data
```

Comment:
You can omit `:data` because it will be inherited from the preceding sections (`.data`)

Reply:
👍🏻